modules/nlu/src/config.ts

```diff
@@ -30,8 +15,15 @@ export interface Config {
   ducklingURL: string
 
   /**
-   * Duckling
+   * Whether or not
    * @default true
    */
   ducklingEnabled: boolean
+
+  /**
+   * The interval at which to automatically sync the models in the background
+   * Set this value to "false" to disable background sync
+   * @default 30s
+   */
+  autoSyncInterval: string
```

Comment:
we should rename this for autoTrainInterval. Sync is quite a weird naming, it assumes that models are trained somewhere else